src/main/java/com/google/devtools/build/lib/packages/StarlarkLibrary.java

```diff
@@ -330,8 +330,10 @@ public Depset depset(
               name = "x",
               positional = true,
               doc =
-                  "A dict that maps configuration conditions to values. Each key is a label string"
-                      + " that identifies a config_setting instance."),
+                  "A dict that maps configuration conditions to values. Each key is a "
```

Comment:
I agree this is a nice PR!

My one quibble is documentation that says "A may be X or Y" without differentiating details is fundamentally ambiguous.

You explain the differentiation particularly well in `macros.md`. I wonder if a cross-reference there would help this.

Reply:
Thanks for the suggestion. I created #14458 as a follow-up, could you take a look?